projects/components/src/data-exporter/csv-exporter.service.spec.ts

```diff
@@ -40,6 +55,13 @@ describe('CsvExporterService', () => {
             const service: CsvExporterService = TestBed.get(CsvExporterService);
             expect(service.createCsv([['a', 'b'], [null, undefined]])).toEqual('a,b\n,');
         });
+
+        it('adds a tab character if the value contains any special characters', () => {
+            const service: CsvExporterService = TestBed.get(CsvExporterService);
+            expect(service.createCsv([['+', '-', '@', '=', 'a+'], [null, undefined]], true)).toEqual(
+                '\t+,\t-,\t@,\t=,a+\n,'
+            );
+        });
     });
```

Comment:
Can we split this into a separate test making sure it doesn't sanitize if control characters are not at the beginning of string. Let's always try to have separate tests for different conditions instead of testing two different things in the same `it()`. The rule is that the text of the it should match the assertion, here there's no it explaining that it does not attempt to sanitize a  control character that is not the first one, but you do seem to be asserting it.